frontends/ol-util/src/components/PageBanner.tsx

```diff
@@ -51,7 +51,6 @@ const imageWrapperHeight = css<ImgProps>`
 const BannerContainer = styled.div`
   position: absolute;
   width: 100%;
-  z-index: -1;
```

Comment:
This was causing issues with click events in the field page and search page (after my change to use BannerPage).

I checked the existing usages, and removing the z-index didn't seem to have any negative affects.